### ISSUE TITLE
Explanation of Scrollbar settings for macOS

### ISSUE THREAD

**Issue by sumyungguy** [piroor/treestyletab#1680]
In the Appearance preferences there's an explanation about macOS and "Overlay":
![screenshot_239](https://user-images.githubusercontent.com/5410893/33955578-49076170-e03c-11e7-87a8-379a4b150306.png)

But this is not true if the user is using a mouse, or has changed the default setting of "Show scroll bars" in System Preferences -> General. In that case, "Default" should be used, and also "Narrow" or "Hide" can be used. This is the case for macOS since 2011 "OS X Lion".

"Default" setting in TST should be used when "Show scroll bars" is set to:
- "Automatically based on mouse or trackpad" (default) and a mouse is used.
- "Always"
- "Always" only for Firefox (affects all scrollbars in Firefox), using the Terminal command:
`defaults write org.mozilla.firefox AppleShowScrollBars -string Always`

In this case, the "Narrow" and "Hide" settings also work.

"Overlay" setting in TST must be used when "Show scroll bars" is set to:
- "Automatically based on mouse or trackpad" (default) and a trackpad is used.
- "When scrolling"

In this case, the other settings don't work correctly.

In other words, if someone using a trackpad wants to hide the scrollbar, they first must change the macOS setting to "Show scroll bars: Always" or use the Terminal command above.

A short explanation in the TST Preferences could be:
> *For macOS, choose "Overlay" when using a trackpad or with "Show scroll bars: When scrolling" in System Preferences. Choose "Default", "Narrow", or "Hide" when using a mouse or with "Show scroll bars: Always".

**Comment by piroor** [comment]
Thanks for detailed explanation! Could you create a pull request for this change? You can create PR on the Web interface, via: https://github.com/piroor/treestyletab/edit/master/webextensions/_locales/en/messages.json

**Comment by sumyungguy** [comment]
Ok, that may take a little while. In the meantime, I release the above text to the public domain. Please feel free to do anything you want with it, no attribution is required.

**Comment by piroor** [comment]
I close this because outdated.
